src/Breadcrumb/Breadcrumb.stories.jsx

```diff
@@ -1,10 +1,5 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import centered from '@storybook/addon-centered';
-import { checkA11y } from '@storybook/addon-a11y';
-import { withInfo } from '@storybook/addon-info';
-import { withReadme } from 'storybook-readme';
```

Comment:
@abutterworth why are these being removed? Because of the Storybook version update?

Reply:
We had configured the same storybook decorators on each story in the past. I don’t know if it’s new with the update but you can set the default configuration for all of them in the storybook config.